lib/resources/virtualAccounts.js

```diff
@@ -66,12 +66,10 @@ module.exports = function (api) {
         return Promise.reject(ID_REQUIRED_MSG);
       }
 
-      let data = {
-        status: 'closed'
-      }
+      let data = {}
```

Comment:
**suggestion**: we can remove this variable itself if we're not sending any data in the request body?

Reply:
i remove that data variable [changes](https://github.com/razorpay/razorpay-node/blob/issue_308/lib/resources/virtualAccounts.js#L70)